### PR TITLE
[Merged by Bors] - Fix Code of Conduct bolding in readme

### PR DESCRIPTION
**Pull request by cart**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Bevy is still in the _very_ early stages of development. APIs can and will chang
 
 ## Community
 
-Before contributing or participating in discussions with the community, you should familiarize yourself with our **[Code of Conduct](./CODE_OF_CONDUCT.md).
+Before contributing or participating in discussions with the community, you should familiarize yourself with our [**Code of Conduct**](./CODE_OF_CONDUCT.md).
```
Comment (james7132) [inline README.md:41]: Should this be inside the link or around the link?

**Review by james7132 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (cart) [reply README.md:41]: Functionally, they are the same. Semantically, I like the current phrasing better. "I want this bold text to be a link" makes more sense to me than "I want this link to be bold".

**Comment by cart** [comment]
Other people merging my PRs is a very cool and very surreal experience :)

**Pull request closed without merging**